packages/odyssey-storybook/package.json

```diff
@@ -34,17 +38,14 @@
     "@storybook/jest": "^0.1.0",
     "@storybook/react": "^7.0.8",
     "@storybook/react-vite": "^7.0.8",
+    "@storybook/test-runner": "^0.10.0",
+    "@storybook/testing-library": "^0.1.0",
     "@storybook/theming": "^7.0.8",
     "@types/mdx": "^2.0.1",
     "@types/react": "^17.0.11",
     "@types/react-dom": "^17.0.2",
-    "@typescript-eslint/eslint-plugin": "^5.57.1",
-    "@typescript-eslint/parser": "^5.57.1",
```

Comment:
We don't need this anymore?

Reply:
Hmm... I don't like the idea of having a global linter, but you're right, we don't have one in Storybook.

That's fine then. We'll (Odyssey person) have to fix it later.